src/TorchVision/Utils.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using static TorchSharp.torch;
+using static TorchSharp.torchvision.io;
+
+namespace TorchSharp
+{
+    public static partial class torchvision
+    {
+        public static class utils
+        {
+            public static Tensor make_grid(
```

Comment:
Should have mentioned this earlier -- doc comments, please!

I usually cut-and-paste from the PyTorch docs, capitalizing the first word of the text (since it seems to be PT convention not to).